binder/requirements.txt

```
@@ -1,4 +1,4 @@
-awkward==1.3.0
-hist[plot]==2.3.0
+awkward>=1.8.0
+hist[plot]~=2.6.0
```

```suggestion
awkward==1.8.0
hist[plot]==2.6.1
```

For reproducibility we want people to pin their imports. The goal is to essentially have people treat their talk like applications (suggesting lock files is something that is probably too advanced for most users so I'm just using `==` as a poor substitute).

---

Fair enough.